src/App/App.test.js

```diff
@@ -1,13 +1,13 @@
 import { render, screen, fireEvent, waitFor } from '@testing-library/react';
-import userEvent from '@testing-library/user-event'
+import userEvent from '@testing-library/user-event';
```

Comment:
Did we ever end up using userEvent?

Reply:
Yes, we use it in one of the tests in this file.